packages/experimental-package/src/global/styles/_carbon-imports.scss

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// @import './carbon-settings';
+
+// Global styles
+// @import 'carbon-components/scss/globals/grid/grid';
+
+// Carbon components
+@import 'carbon-components/scss/components/button/button';
```

Comment:
I wonder if this import should live within the Sass files of `ExampleComponent` if it's being explicitly used by that?

Reply:
That should be OK as Carbon builds in a use once mechanism.

A note of caution perhaps with regards to including SCSS imports in the react component. Some webpack builds treat components as separate compilation units which results in many copies of SCSS prior to any minification.